Formula/tweak.rb

```diff
@@ -0,0 +1,20 @@
+class Tweak < Formula
+  desc "Command-line, ncurses library based hex editor"
+  homepage "https://www.chiark.greenend.org.uk/~sgtatham/tweak/"
+  url "https://www.chiark.greenend.org.uk/~sgtatham/tweak/tweak-3.02.tar.gz"
+  sha256 "5b4c19b1bf8734d1623e723644b8da58150b882efa9f23bbe797c3922f295a1a"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "make"
+    bin.install "tweak"
+    man1.install "tweak.1"
+    doc.install "btree.html"
```

Comment:
`system "make", "install"`

Reply:
i tried make install both in the brew install --interactive tweak and by making the file say that then running brew install tweak and it broke, saying cannot install into /usr/local/bin/tweak ... I think it's because the source .tar.gz doesn't have a ./configure?

Reply:
`system "make", "install", "PREFIX=#{prefix}"`